### PR TITLE
docs: [no-floating-promises] add MDN link regarding void operator

### PR DESCRIPTION
**Pull request by kirkwaiblinger**
## PR Checklist

- [ ] Addresses an existing open issue: fixes #000
- [ ] That issue was marked as [accepting prs](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
- [x] Steps in [Contributing](https://typescript-eslint.io/contributing) were taken

### CODE REVIEW

File: packages/eslint-plugin/docs/rules/no-floating-promises.mdx [hunk 1]
```diff
@@ -131,7 +131,7 @@ await createMyThenable();
 
 ### `ignoreVoid`
 
-This option, which is `true` by default, allows you to stop the rule reporting promises consumed with void operator.
+This option, which is `true` by default, allows you to stop the rule reporting promises consumed with the [void operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void).
```
Comment (JoshuaKGoldberg) [inline packages/eslint-plugin/docs/rules/no-floating-promises.mdx:134]:
```suggestion
This option, which is `true` by default, allows you to stop the rule reporting promises consumed with the [`void` operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void).
```

**Review by JoshuaKGoldberg — DISMISSED** [review]
i like

File: packages/eslint-plugin/docs/rules/no-floating-promises.mdx (hunk 1 above)
Comment (kirkwaiblinger) [reply packages/eslint-plugin/docs/rules/no-floating-promises.mdx:134]: Oh, yeah, meant to do that

**Review by bradzacher — APPROVED** [review]

**Comment by Josh-Cena** [comment]
We should definitely do this but I wonder if it can just be coalesced into https://github.com/typescript-eslint/typescript-eslint/pull/9949?

**Review by JoshuaKGoldberg — APPROVED** [review]

**Comment by bradzacher** [comment]
cos it's so simple let's just get this in cos it's nice right now.

**Pull request merged**